models/comment.rb

```diff
@@ -24,6 +24,7 @@ class Comment < Content
 
   index({author_id: 1, course_id: 1})
   index({_type: 1, comment_thread_id: 1, author_id: 1, updated_at: 1})
+  index({comment_thread_id: 1, author_id: 1, created_at: 1})
```

Comment:
Do you know why type is on all the original indexes and what the actual query works out to be?